pkg/conduit/config.go

```diff
@@ -72,6 +81,21 @@ func (c Config) Validate() error {
 		return requiredConfigFieldErr("http.address")
 	}
 
+	if c.Log.Level == "" {
+		return requiredConfigFieldErr("log.level")
+	}
+	levels := map[string]int{"debug": 1, "info": 2, "warn": 3, "error": 4, "trace": 5}
```

Comment:
yep exactly, it was just a way to make a quick search using a map (rather than looping through a slice). any other suggestions to do this?

Reply:
My 2 cents - try to run [`zerolog.ParseLevel`](https://pkg.go.dev/github.com/rs/zerolog#ParseLevel) and see if it returns an error. I know this will mean we support more than just the 5 log levels we talked about, but I see no harm in that.